server/service/individualServices/IndividualServicesOperationsMapping.js

```diff
@@ -0,0 +1,9 @@
+module.exports.individualServicesOperationsMapping =
+{
+  "/v1/redirect-info-about-approval-status-changes": {
+    "subscriber-operation": {
```

Comment:
These changes are not part of this pull request.please remove them.

Reply:
done